docs/network/evm-nodes.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Running an Etherlink EVM node
+---
+
+The Etherlink EVM nodes are responsible for maintaining a copy of the Etherlink context and applying new blueprints that process EVM transactions.
+
+## Prerequisites
+
+- Make sure you understand the interaction between different nodes as described in [Etherlink architecture](./architecture).
+- Run an Etherlink Smart Rollup node as described in [Running an Etherlink Smart Rollup node](./smart-rollup-nodes).
+Public Smart Rollup nodes for Etherlink are not yet available, so you must run your own if you want to participate in the Etherlink network.
+- Get the Etherlink installer kernel (`installer.hex` file), which you can build yourself as described in [Building the Etherlink kernel](./building-kernel) or download here: [installer.hex](/files/installer.hex).
+
+## Running an Etherlink EVM node
```

Comment:
Now that we have smart rollup snapshot we can start an Etherlink EVM node without the initial kernel.
But maybe in another MR ?

Reply:
with the `init from rollup node` command

Reply:
That's new to me. I'll make a note to handle that in a future PR.

Reply:
Do you uses issue for doc PR ?
if yes, could you turn this into a follow-up ?

Reply:
Ok, here's the issue for starting the EVM node without the initial kernel: https://github.com/etherlinkcom/docs/issues/63